README.md

```diff
@@ -337,6 +337,7 @@ package cmd
 
 import (
 	"github.com/spf13/cobra"
+	"fmt"
 )
```

Comment:
Why not do this similar to https://github.com/spf13/cobra#prerun-or-postrun-hooks, where `fmt` comes first, blank line, then `cobra`?